docker-compose.yml

```diff
@@ -3,76 +3,111 @@ version: "3.9"
 # First draft 
 services:
   # ────────────────────────────
-  backend:
+  prometheus_app:
```

Comment:
bad name : this is not a prometheus application. It's an API monitor by prometheus. Maybe `core_status_api` is better